packages/common/src/hooks/useDetailPanel.tsx

```diff
@@ -49,6 +49,7 @@ export const useDetailPanel = (): DetailPanel => {
         icon={<MenuDots />}
         labelKey="button.more"
         onClick={() => open()}
+        shouldShrink
```

Comment:
leftover issue to address, from [this comment](https://github.com/openmsupply/openmsupply-client/pull/170#pullrequestreview-757936443)